R/plotPValHist.R

```diff
@@ -4,7 +4,8 @@
 #' numbers. Intended to perform histogram analysis of p-value distributions,
 #' but should be useful for any dataframe of numeric columns.
 #'
-#' @param P.Val A matrix or dataframe of numeric data; col = samples
+#' @param DGEdata Name of DGEobj with a class of DGEobj.
```

Comment:
I do not think that is an accurate documentation. As user do not pass the object name she/he pass the object itself.

Reply:
done